strap.py

```
@@ -1,22 +1,53 @@
         url = i.split(" - ")[1]
         print("downloading " + url)
-        if not os.path.exists("strap_packages/" + package + ".tar.zst"):
-            os.system("curl -# -L -o strap_packages/" + package + ".tar.zst " + url)
-        print("installing " + package)
-        os.system(
-            "tar -I zstd -xf strap_packages/"
-            + package
-            + ".tar.zst -C rootfs/filesystem --strip-components=1"
+        tarball = "strap_packages/" + package + ".tar.zst"
+        if not os.path.exists(tarball):
+            os.system("curl -# -L -o " + tarball + " " + url)
```

Ruff (0.15.12)

[error] Starting a process with a shell, possible injection detected

(S605)